src/olympia/abuse/migrations/0041_alter_decision_date.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.16 on 2024-10-14 13:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Would be nice to have a reversible migration here. It's not clear what the field "used" to be and if you ever wanted to undo the migration this would be useful.. possible?

Reply:
built-in django migrations are reversible by default.

Reply:
TIL.